sympy/core/operations.py

```diff
@@ -32,7 +34,11 @@ class AssocOp(Basic):
     @cacheit
     def __new__(cls, *args, **options):
```

Comment:
Would it be better to add
```suggestion
    def __new__(cls, *args, sympify=True,  **options):
```
here? That way, the existence of the argument is visible from the output of `help`